NDcPP_v2_2e.adoc

```diff
@@ -1932,12 +1933,17 @@ _The use of X.509v3 certificates for TLS is addressed in FIA_X509_EXT.2.1. This
 
 *FCS_TLSS_EXT.2 TLS Server Support for Mutual Authentication*
 
-*FCS_TLSS_EXT.2.1* The TSF shall support TLS communication with mutual authentication of TLS clients using X.509v3 certificates.
+*FCS_TLSS_EXT.2.1* The TSF shall support TLS communication with mutual authentication of TLS clients using X.509v3 certificates and shall [selection:
+
+* _reject the connection if the client does either not provide a client certificate at all or the client certificate cannot be successfully validated by the TOE (except for override mechanisms that might be defined in FCS_TLSS_EXT.2.2) ('hard fail')_
```

Comment:
Please change to:  "...reject the connection if the client either does not provide a client certificate at all, or the client certificate..."

Reply:
@plughy2 Good catch on the wording. I've made that change here and in the word doc.